submitted_models/marble_hd2_sensor_config_1/specifications.md

```diff
@@ -70,15 +83,16 @@ Computers were installed in the payload area of the husky and these have been ro
   * RPLidar S1 Planar Lidar - https://www.slamtec.com/en/Lidar/S1Spec
   * Vividia HTI-301 LWIR Camera - https://www.oasisscientific.com/store/p504/Vividia_HTi_HT-301_Thermal_Imaging_Camera_for_Android_Phone_and_Tablet_with_IR_Resolution_384x288.html  
   * IMU: Microstrain 3DM-GX5-25 - datasheet: https://www.microstrain.com/sites/default/files/applications/files/3dm-gx5-25_datasheet_8400-0093_rev_n.pdf
-    * Explanation of sensor parameter derivations:
+    ~~* Explanation of sensor parameter derivations:
 	We derived the stddev terms as follows:
 
 	accelerometer noise density = 0.00002 g/sqrt(Hz) 
 		=> convert to m/s^2 => 1.962e-4 m/s^2
 	gyro noise density = 0.005 deg/s/sqrt(Hz)
 		=> convert to rad/sec => 8.72664e-5 radians
 
-	Other terms are difficult to extract from datasheet, so we used similar terms to previous IMU models proposed (of similar or worse quality) such as the ADIS 16448 (which has worse performance than this IMU). 
+	Other terms are difficult to extract from datasheet, so we used similar terms to previous IMU models proposed (of similar or worse quality) such as the ADIS 16448 (which has worse performance than this IMU). ~~
+    - IMU model is standardized to match other available models.
 
 * \<Validation Video Link(s), e.g.,https://youtu.be/xxxxxxxxx/\>
```

Comment:
Are these placeholders for Validation links still supposed to be here?

Reply:
Yes. We expect a later PR submitting the validation data and updating the MARBLE models' velocity, accel, and endurance limits to match.